modules/claims_api/spec/sidekiq/poa_form_builder_job_spec.rb

```diff
@@ -170,4 +171,24 @@
       end
     end
   end
+
+  context 'when the BD upload feature flag is enabled' do
+    it 'uploads to the benefits document API' do
+      Flipper.enable(:lighthouse_claims_api_poa_use_bd)
+      expect(ClaimsApi::VBMSUploader).not_to receive(:new)
+      expect_any_instance_of(ClaimsApi::VBMSUploader).not_to receive(:upload_document)
+      power_of_attorney.update(form_data: power_of_attorney.form_data.deep_merge(
+        {
+          docType: 'L190'
+        }
```

Comment:
From the task description:

> Uploads of VA Form 21-22, Appointment of Veterans Service Organization as Claimant's Representative, will include docType of L190

Can we assume the `docType` will be at the top level of the request body?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17587/commits/987270ad204d288917a1149f4bdc0c476e9b3749